_includes/head.html

```diff
@@ -2,7 +2,7 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700" rel="stylesheet" type="text/css">
-    <link rel="stylesheet" href="{{ site.url }}{{ site.baseurl }}/style/style.css">
+    <link rel="stylesheet" href="/style/style.css">
```

Comment:
Don't want to accidentally commit this to `master`.

Reply:
This is from `master`. It's in production successfully now… 🤷‍♂️